CMakeLists.txt

```diff
@@ -80,6 +80,8 @@ add_definitions(-fPIC)
 #
 IF (${DISTRO} STREQUAL "Ubuntu_17.04")
  set (CMAKE_CXX_FLAGS "-Wall -Wno-unknown-pragmas -D_LARGEFILE64_SOURCE -D_GNU_SOURCE -Werror -Wno-deprecated-declarations -fno-var-tracking-assignments")
+ELSEIF (${DISTRO} STREQUAL "Ubuntu_17.10")
+ set (CMAKE_CXX_FLAGS "-Wall -Wno-unknown-pragmas -D_LARGEFILE64_SOURCE -D_GNU_SOURCE -Werror -Wno-deprecated-declarations -fno-var-tracking-assignments")
```

Comment:
This set if the same than the one for 17.04. Maybe you could generalize boths in the same clause (looking for "Ubuntu_17.*" but in the right CMakeList.txt syntax, of course).

Or at least using an AND for joining both STREQUAL clauses.

Probably there are examples of both in existing CMakeLists.txt files.

Reply:
Fixed in 3cb7510